algorithms/pact/pact_ops.py

```diff
@@ -1433,8 +1430,27 @@ def get_eps_out(self, eps_in):
 
 
 class PACTIntegerHardswish(nn.Module):
-    def __init__(self, eps_in : float, eps_s : float):
+
+    class MyHardswish(torch.autograd.Function):
```

Comment:
Please make sure these changes are compatible with @da-gazzi's work on MBNv3

Reply:
After talking with @Victor-Jung today, Georg is on (well-earned) vacation, so let's not hold this up. Worst case we can revert or amend if this is incompatible with Georg's work.